### PR TITLE
Feat/main layout

### PR DESCRIPTION
**Pull request by menyaboo**

### CODE REVIEW

File: src/entities/container-box/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { ContainerBoxStyled } from '@/entities/container-box/style.ts'
+
```
Comment (AziAlex) [inline src/entities/container-box/index.tsx:2]: убери тип Props

File: src/entities/container-box/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+type Props = {
+  children: React.ReactNode
+}
+
```
Comment (AziAlex) [inline src/entities/container-box/index.tsx:6]: rewrite it like this:
export const ContainerBox: FC<PropsWithChildren> = ({children})

File: src/pages/main-page/index.tsx [hunk 3]
```diff
@@ -1,7 +1,30 @@
 import { NavAccessories } from '@/widgets/nav-accessories'
+import { ContainerBox } from '@/entities/container-box'
+import styled from 'styled-components'
+
```
Comment (AziAlex) [inline src/pages/main-page/index.tsx:4]: Containers with styles must be transferred to the styles.ts file

File: src/pages/main-page/index.tsx [hunk 4]
```diff
@@ -1,7 +1,30 @@
...
+`
 
 export const MainPage = () => (
-  <>
+  <MainPageWrapper>
     <NavAccessories />
-  </>
-)
\ No newline at end of file
+    <GridContainersBoxes>
```
Comment (AziAlex) [inline src/pages/main-page/index.tsx:22]: image
image

File: src/entities/container-box/index.tsx
Comment (AziAlex) [inline src/entities/container-box/index.tsx:1]: this component is not an entity but a widget

**Review by AziAlex — COMMENTED** [review]
fix it

File: src/entities/container-box/index.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { ContainerBoxStyled } from '@/entities/container-box/style.ts'
+
+type Props = {
+  children: React.ReactNode
+}
```
Comment (AziAlex) [inline src/entities/container-box/index.tsx]: delete type Props

File: src/entities/container-box/index.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+export const ContainerBox = (props: Props) => (
+  <ContainerBoxStyled>{props.children}</ContainerBoxStyled>
+)
```
Comment (AziAlex) [inline src/entities/container-box/index.tsx]: rewrite it like this:
image

File: src/pages/main-page/index.tsx [hunk 7]
```diff
@@ -1,7 +1,30 @@
...
+
+export const MainPageWrapper = styled.div`
+  height: 100%;
+  display: flex;
+  gap: 40px;
+`
+
+export const GridContainersBoxes = styled.div`
+  display: grid;
+  width: 100%;
```
Comment (AziAlex) [inline src/pages/main-page/index.tsx]: Containers with styles must be transferred to the styles.ts file

File: src/pages/main-page/index.tsx [hunk 8]
```diff
@@ -1,7 +1,30 @@
...
-  <>
+  <MainPageWrapper>
     <NavAccessories />
-  </>
-)
\ No newline at end of file
+    <GridContainersBoxes>
+      {[...Array(4)].map(() => (
+        <ContainerBox>
+          <></>
```
Comment (AziAlex) [inline src/pages/main-page/index.tsx]: 295166407-ebbfbfa2-157a-4db8-bf18-5d0aa5a17b42
295166013-f6ac111c-2a67-49b8-b1ea-db54b6c02a6a

File: src/entities/container-box/index.tsx (hunk 5 above)
Comment (menyaboo) [reply src/entities/container-box/index.tsx]: add custom styles types from Props
example: liner-gradient, pading stules...

**Review by AziAlex — APPROVED** [review]

**Pull request merged**